src/content/peerconnection/endtoend-encryption/js/videopipe.js

```diff
@@ -58,44 +41,38 @@ function VideoPipe(stream, sendTransform, receiveTransform, handler) {
         .pipeThrough(senderTransformStream)
         .pipeTo(senderStreams.writableStream);
   }
-  pc1.onicecandidate = function(event) {
-    if (event.candidate) {
-      pc2.addIceCandidate(new RTCIceCandidate(event.candidate),
-          noAction, errorHandler('AddIceCandidate'));
-    }
-  };
-  pc2.onicecandidate = function(event) {
-    if (event.candidate) {
-      pc1.addIceCandidate(new RTCIceCandidate(event.candidate),
-          noAction, errorHandler('AddIceCandidate'));
-    }
-  };
-  pc2.ontrack = function(e) {
+  this.pc2.ontrack = e => {
     if (receiveTransform) {
       const transform = new TransformStream({
         start() {},
```

Comment:
Nit: These are optional (`start` and `flush`)

Reply:
wrong PR for nitting on that :-)